tweet_recommendations/validators/graph_summarization_validator.py

```diff
@@ -12,10 +14,28 @@
 
 tweets_with_lemmas = lemmatizer.fit_transform(original_tweets, verbose=True)
 the_graph.fit(tweets_with_lemmas, minimal_hashtag_occurence=3)
-print(the_graph.transform(("Wiozę szwagra na wybory",))[0][:10])
-print(the_graph.transform(("Co by tu dzisiaj zrobić ...",))[0][:10])
-print(the_graph.transform((
-    "Co by tu dzisiaj zrobić ...",
-    "Wiozę szwagra na wybory",
-    "Co by tu dzisiaj zrobić ...",
-), query=["StrajkLOT", "Jesień", "Starogard"])[:, :10])
+# print(the_graph.transform(("Wiozę szwagra na wybory",))[0][:10])
```

Comment:
Ja bym to wywalił, te testy szybkości, nie wiem czy chcemy mieć to na masterze @kacper1095 ale nie mam jakiegoś mocnego zdania na ten temat :)